contracts/staking/FloatingStakingPool.sol

```diff
@@ -42,6 +42,8 @@ contract FloatingStakingPool is StakingPoolBase {
 
         stakingPoolInfo.totalStakedAmount += uint128(amount);
 
+        emit Deposit(_msgSender(), amount);
```

Comment:
question: did you figure out a rule or guide for when we should output the account that triggered a state change?

If so, then please add deets to https://github.com/windranger-io/windranger-treasury/issues/245 🙂